src/js/pages/KanbanBoard/components/RecordCommentsComponents/RecordCommentsModal.js

```diff
@@ -82,6 +82,7 @@ class RecordCommentModal extends Component {
   saveComment = () => {
     const {id, type, chunkId, chunkNum} = this.props;
     const {jsonBlob} = this.state;
+    //console.log(jsonBlob,id, chunkId, chunkNum,type, 'data for comment creation');

```

Comment:
Here is a console.log as a comment.

Reply:
so i noticed those while i was working on this test branch. I'll remove them from the Pr real quick